### PR TITLE
feat: strategy variant migrations

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes

Adding variants JSONB column to strategies that will allow subsequent PRs introduce support for the strategy variants. 
It's less than 10 LOC, most of the code is the old view definition with a new line added.
More comments inline.

### CODE REVIEW

File: src/migrations/20230712091834-strategy-variants.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+          ALTER TABLE feature_strategies ADD COLUMN IF NOT EXISTS variants JSONB DEFAULT '[]'::jsonb NOT NULL;
```
Comment (kwasniew) [inline src/migrations/20230712091834-strategy-variants.js:6]: new column

File: src/migrations/20230712091834-strategy-variants.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+              environments.sort_order as environment_sort_order,
+              feature_strategies.id as strategy_id,
+              feature_strategies.strategy_name as strategy_name,
+              feature_strategies.parameters as parameters,
+              feature_strategies.constraints as constraints,
+              feature_strategies.sort_order as sort_order,
+              fss.segment_id as segments,
+              feature_strategies.title as strategy_title,
+              feature_strategies.disabled as strategy_disabled,
+              feature_strategies.variants as strategy_variants
```
Comment (kwasniew) [inline src/migrations/20230712091834-strategy-variants.js:33]: this is the only new line compared to the previous view
Screenshot 2023-07-13 at 08 20 11

File: src/migrations/20230712091834-strategy-variants.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+              environments.type as environment_type,
+              environments.sort_order as environment_sort_order,
+              feature_strategies.id as strategy_id,
+              feature_strategies.strategy_name as strategy_name,
+              feature_strategies.parameters as parameters,
+              feature_strategies.constraints as constraints,
+              feature_strategies.sort_order as sort_order,
+              fss.segment_id as segments,
+              feature_strategies.title as strategy_title,
+              feature_strategies.disabled as strategy_disabled
```
Comment (kwasniew) [inline src/migrations/20230712091834-strategy-variants.js:77]: this is the only new line compared to the previous rollback code
Screenshot 2023-07-13 at 08 20 15

**Review by andreas-unleash — APPROVED** [review]
LG

**Pull request merged**